pkg/vetter/util/mtlspolicy/authPolicy_test.go

```diff
@@ -160,56 +193,475 @@ var (
 var _ = Describe("LoadAuthPolicies", func() {
 	It("should load policies", func() {
 		loaded, err := LoadAuthPolicies([]*authv1alpha1.Policy{
-			apDefaultOn,
-			apFooOn,
-			apFooOff,
-			apFooBarOn,
-			apFooPortsBarOn,
+			nsbarNs_On,
+			nsDefault_apFoo_On,
+			nsDefault_apFoo_Off,
+			nsDefault_apFoo_apBar_On,
+			nsDefault_apFooPorts_apBar_On,
 		})
-		Expect(err).To(Succeed())
+		Expect(err).To(BeNil())
+		loaded.LoadMeshPolicy([]*authv1alpha1.MeshPolicy{meshDefaultOn})
```

Comment:
You're still working on the change where LoadAuthPolicies() takes both authv1alpha1.Policy and authv1alpha1.MeshPolicy, right?